UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/appsettings.json

```diff
@@ -58,6 +58,13 @@
         "ProductFileName": "PRODUCTS.TXT",
         "Info": "INFO"
     },
+    "PKSApiConfiguration": {
+        "BaseUrl": "",
+        "ClientId": "",
+        "ClientSecret": "",
+        "AccessTokenUrl": "",
```

Comment:
Instead of AccessTokenURL in appsettings, we can add MicrosoftOnlineLoginUrl and concat the access URL in code.

So appsettings for PKS configuartion would like-
```
"PKSApiConfiguration": {
    "BaseUrl": "",
    "MicrosoftOnlineLoginUrl": "https://login.microsoftonline.com/",
    "TenantId": ""   
    "ClientId": "",
    "ClientSecret": ""  
},
```

With this change, even devops would require to modify the PKS configuration settings

Reply:
Resolved